priority/priority.erl

```diff
@@ -1,47 +1,45 @@
 -module(priority).
+
 -export([start/0,
          send_vip/2,
          send_normal/2]).
+
  -include_lib("eunit/include/eunit.hrl").
 
+start() ->
+    spawn(fun() -> priority_loop() end).
```

Comment:
you can use this syntax instead `fun priority_loop/0`, since you're just calling a named function